README.md

```diff
@@ -40,22 +40,24 @@ In this tutorial you will:
 
 ## Data
 
-* [Cholera deaths](https://github.com/mjdanielson/Cholera-Map/blob/master/Data/Cholera_Deaths.geojson): Number of deaths attributed to cholera in 1854 by location. Data sourced by [Robin Wilson at Southampton] University(http://blog.rtwilson.com/john-snows-cholera-data-in-more-formats/).
+Save the following **GeoJSON** files to your computer:
 
-* [Location of pumps](https://github.com/mjdanielson/Cholera-Map/blob/master/Data/Cholera_Pumps.geojson)
+* [Cholera deaths](https://raw.githubusercontent.com/mjdanielson/Cholera-Map/master/Data/Cholera_Deaths.geojson): Number of deaths attributed to cholera in 1854 by location. Data sourced by [Robin Wilson at Southampton University](http://blog.rtwilson.com/john-snows-cholera-data-in-more-formats/).
```

Comment:
changed the geojson links to point to the *raw* version of the file so you can just right-click > save as.